src/main/java/com/base64/d/WriteToFile.java

```diff
@@ -3,51 +3,27 @@
 import java.io.FileWriter;
 import java.io.IOException;
 import java.io.PrintWriter;
-import java.io.*;
-import java.nio.file.Files;
-import java.nio.file.Paths;
-import java.nio.file.StandardOpenOption;
-import java.text.SimpleDateFormat;
 import java.util.Date;
 
-import static java.lang.System.out;
-
-
 public class WriteToFile {
 
-    String date = new Date().toString();
```

Comment:
By defining the object here, you are specifying an instance variable. This will be set for each instance of `WriteToFile`. You typically want to push variables into the smallest scope possible. The scope going from "class" to "instance" to "method". So I made this a method level variable by placing it inside the method. This keeps the variables as close to the place that they are used.